tests/unit/preprocessor/_regrid/test_extract_levels.py

```diff
@@ -145,8 +149,15 @@ def test_interpolation__masked(self):
         masked = ma.empty(self.shape)
         masked.mask = mask
         cube = _make_cube(masked, dtype=self.dtype)
+        # save cube to test the lazy data interpolation too
+        iris.save(cube, self.filename)
         with mock.patch(
                 'stratify.interpolate', return_value=new_data) as mocker:
+            # first test lazy
+            loaded_cube = iris.load_cube(self.filename)
```

Comment:
To make a cube with realized data lazy, you can do
```
cube.data = cube.lazy_data()
```
or
```
cube.data = da.asarray(cube.data, chunks=(1, 2, 3, 4))
```
if you want more control over how the lazy array is created. There is no need to first save to file.

Reply:
cool! I didn't know that, but I wanted to replicate the actual conditions the function runs in anyway